### ISSUE TITLE
Keep the Rigify metarig around

### ISSUE THREAD

**Issue by toshicg** [soupday/cc_blender_tools#110]
The ActorCore to Rigify quick rig function is great.
I just wish it would keep the Metarig in the scene so that you can tweak it. 
The main reason is that sometimes the fingers are not generated correctly so I want to go back an tweak their orientation a tiny bit.

**Comment by soupday** [comment]
The advanced setup mode does it in stages so you can attach and tweak the meta rig before rigifying it.

**Comment by soupday** [comment]
But it doesn't keep the meta rig to refine it...

**Comment by toshicg** [comment]
> But it doesn't keep the meta rig to refine it...

Exactly. Is there a chance to make so that the metarig stays after generation for possible tweaking?
